### PR TITLE
Fix #952 - Update HREF link to Relay add-on download page

### LINKED ISSUE #952: Home page: "Download the ⁨Relay⁩ extension for ⁨Firefox" link broken
STR:

1. Go to relay.firefox.com
2. Scroll to "How It Works"
3. Click the "Download the ⁨Relay⁩ extension for ⁨Firefox" under the "Install the extension" section

Expected results:
Get to https://addons.mozilla.org/en-US/firefox/addon/private-relay/

Actual results:
404 at https://relay.firefox.com/%E2%81%A8https://addons.mozilla.org/firefox/addon/private-relay/%E2%81%A9

### PR DESCRIPTION
**Pull request by maxxcrawford**

### CODE REVIEW

File: privaterelay/locales/en-US/app.ftl [hunk 1]
```diff
@@ -94,20 +90,20 @@ hero-image-copy-control-html = Now <em>you’re in control</em> of what’s arri
...
+faq-question-1-answer-a = While { -brand-name-relay } does not filter for spam, our email partner Amazon SES does block spam and malware. If { -brand-name-relay } forwards messages you don’t want, you can update your { -brand-name-relay } settings to block messages from the alias forwarding them.  
 
 # Variables:
 #   $url (url) - https://addons.mozilla.org/firefox/addon/private-relay/
 #   $attrs (string) - specific attributes added to external links
-faq-question-1-answer-b-html = If you see a broader problem of unwanted email from all of your aliases, please <a href="{ $url }" { $attrs }>report this to us</a> so we can consider adjusting the SES spam thresholds for this service. If you report these as spam, your email provider will see { -brand-name-relay } as the source of spam, not the original sender.
+faq-question-1-answer-b-html = If you see a broader problem of unwanted email from all of your aliases, please <a href={ $url } { $attrs }>report this to us</a> so we can consider adjusting the SES spam thresholds for this service. If you report these as spam, your email provider will see { -brand-name-relay } as the source of spam, not the original sender.
```
Comment (groovecoder) [inline privaterelay/locales/en-US/app.ftl:98]: Why is this removing the `"` chars?

Comment (maxxcrawford) [reply privaterelay/locales/en-US/app.ftl:98]: The original intent was to make it render correctly (as it was adding quotes already), however – that is not the case. I'll add them back.

File: privaterelay/templates/home.html [hunk 2]
```diff
@@ -36,7 +36,10 @@ <h2 class="how-it-works-header text-center">{% ftlmsg 'how-it-works-headline' %}
             <div class="step-text step-1">
               <h3 class="step-headline ff-Met">{% ftlmsg 'how-it-works-step-1-headline' %}</h3>
               <p class="step-body">
-                {% ftlmsg 'how-it-works-step-1-copy-html' url='https://addons.mozilla.org/firefox/addon/private-relay/' attrs='target="_blank" rel="noopener" class="text-link"' %}
+                <a href="https://addons.mozilla.org/firefox/addon/private-relay/" target="_blank" rel="noopener" class="text-link">
+                  {% ftlmsg 'how-it-works-step-1-link' %}
+                </a>
```
Comment (groovecoder) [inline privaterelay/templates/home.html:41]: This underlines the space after the `.`.
image


```suggestion
                <a href="https://addons.mozilla.org/firefox/addon/private-relay/" target="_blank" rel="noopener" class="text-link">{% ftlmsg 'how-it-works-step-1-link' %}</a>
```

**Review by groovecoder — CHANGES_REQUESTED** [review]

File: privaterelay/templates/home.html (hunk 2 above)
Comment (maxxcrawford) [reply privaterelay/templates/home.html]: Revised! 

image

**Review by groovecoder — APPROVED** [review]

**Pull request merged**